server/Configurations.json

```diff
@@ -0,0 +1,8 @@
+{
+	"hostName": "localhost:8080",
+	"port": "8080",
+	"dataLocalPath": "/home/mkhalil/versioned/cogneco/exclusive/build/data",
+	"appPath": "/home/mkhalil/versioned/cogneco/exclusive/build",
+	"authorisationServer": "imint.highrisehq.com",
+	"authorisationPath": "/me.xml"
+}
```

Comment:
This should not be committed. And you probably do not need full path as dataLocalPath. From project root should be enough.